src/index.js

```diff
@@ -5,6 +5,14 @@ import React, { useState } from "react";
 import { createRoot } from 'react-dom/client';
 import reportWebVitals from "reportWebVitals";
 
+/** removing console statement in react prod build */
+if (process.env.NODE_ENV !== "development") {
+  console.log = () => {};
```

Comment:
I am curious, how do these functions work? Can you please provide documentation/video regarding this?

Reply:
The main concept is - 
* I check this current node environment is   `development` or `production`
* If it's `not` `development`.  I overwrite so of the default console behavior. 

How it works- 
In production, when you call `console.log` statement it will call an empty function. That's it.


`Development` | `Production` |
--- | --- | 
![image](https://user-images.githubusercontent.com/21246862/171913565-179cc65e-5603-497b-9d99-abc84c1d255f.png) | ![image](https://user-images.githubusercontent.com/21246862/171913704-40e8ef5b-5fd5-41db-8dff-cc44b92653d5.png)  |

I think it will help you.